ci-operator/step-registry/sandboxed-containers-operator/README.md

```diff
@@ -0,0 +1,57 @@
+This directory contain the steps, chains and workflows implemented specifically for the Openshift Sandboxed Containers (OSC) jobs.
+
+## Steps
+
+Here is the list of steps and their explanation.
+
+Please refer to the `*-ref.yaml` file in their source code for the full list of parameters accepted by each step.
+
+### sandboxed-containers-operator-get-kata-rpm
+
+The [sandboxed-containers-operator-get-kata-rpm](./get-kata-rpm/) step downloads the kata-containers rpm from Brew and copy it over the cluster worker nodes.
+
+### sandboxed-containers-operator-peerpods-param-cm
+
+The [sandboxed-containers-operator-peerpods-param-cm](./peerpods/param-cm/) step creates the peerpods-param-cm configmap. Currently only Azure is supported and it will do the needed networking setup for OSC to work properly on this cloud provider.
+
+### sandboxed-containers-operator-env-cm
+
+The [sandboxed-containers-operator-env-cm](./env-cm/) step creates the osc-config configmap which is actually used by the OSC tests in `platform-extended-tests` to control many aspects of the execution. In case this step is not reference, default values will be used by the tests.
+
+Currently not all parameters are enabled. In particular, only GA release type is supported, meaning it doesn't install development builds of OSC.
+
+## Chains
+
+Here is the list of chains.
+
+### sandboxed-containers-operator-pre
+
+The [sandboxed-containers-operator-pre](./pre/) chain wraps the steps that prepare the environment for executing the tests.
+
+This chain is meant to be referenced in the `pre` condition of the workflow.
+
+## workflows
+
+Here is the list of workflows.
+
+### sandboxed-containers-operator-e2e-azure
+
+The [sandboxed-containers-operator-e2e-azure](./e2e/azure/) workflow implements an entire e2e execution for testing OSC on Azure. It will deploy Openshift on Azure, evoke the [sandboxed-containers-operator-pre](#sandboxed-containers-operator-pre) chain for preparing the environment and finally execute the `platform-extended-tests`.
+
+As the [openshift-extended-test](../openshift-extended/test/) step is referenced in the `test` condition, any job using this workflow should import the `tests-private` image. This is done by adding an entry to the `base_images` section in the job's yaml, as for example:
+
+```yaml
+base_images:
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.18"
+```
+
+> **Important:** updates to our tests in `platform-extended-tests` are made to `master` and never backported to release branches, however, the `tests-private` image hasn't `latest` builds from the `master` branch. Meaning that if you need to pick the latest and greatest code of `platform-extended-tests` then you must find and use the latest image version available at that point in time (usually it is next major OCP version under development).
```

Comment:
Perhaps it's worth mentioning that one can get the list of tags by: `skopeo list-tags docker://registry.ci.openshift.org/ci/tests-private`